data/registers/lptim_v2b.yaml

```diff
@@ -0,0 +1,728 @@
+block/IC:
+  items:
+  - name: ISR
+    description: LPTIM interrupt and status register.
+    byte_offset: 0
+    fieldset: ISR_IC
+  - name: ICR
+    description: LPTIM interrupt clear register.
+    byte_offset: 4
+    fieldset: ICR_IC
+  - name: DIER
+    description: LPTIM interrupt enable register.
+    byte_offset: 8
+    fieldset: DIER_IC
+block/LPTIM_ADV:
+  extends: LPTIM_BASIC
+  description: Low power timer with Output Compare
+  items:
+  - name: InputCapture
```

Comment:
Instead of doing two overlapping blocks for the two modes like the RM, could you merge the bits in the two modes of the registers? it makes it simpler. 

I don't know why the RM splits it like this, all the bits that are common between the two modes match exactly, there's no "this bit does X in capture mode and Y in compare mode"....